sdk/master_board_sdk/include/master_board_sdk/motor_driver.h

```diff
@@ -18,9 +18,14 @@ class MotorDriver
 
 	void Enable();
 	void Disable();
+
+	//adc
+	float adc[2];
```

Comment:
Is this the proper place to put the adc measurements? They are not really part of the motor, so putting them on the `MotorDriver` class feels a bit off.

How about renaming `MotorDriver` to `MotorBoardDriver` (that widens the scope a bit)?